Tutorial.md

```diff
@@ -11,455 +12,240 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
-                        "self": { "href": "/gists/42/star" },
-                    },
-                    "starred": true
-                }
++ Response 200 (application/json)
 
-    ### Star a Gist [PUT]
-    + Response 204
+            {
+                "question": "Favourite programming language?",
+                "published_at": "2014-11-11T08:40:51.620Z",
+                "url": "/questions/1",
+                "choices": [
+                    {
+                        "answer": "Swift",
+                        "url": "/questions/1/choices/1",
+                        "votes": 2048
+                    }, {
+                        "answer": "Python",
+                        "url": "/questions/1/choices/2",
+                        "votes": 1024
+                    }, {
+                        "answer": "Objective-C",
+                        "url": "/questions/1/choices/3",
+                        "votes": 512
+                    }, {
+                        "answer": "Ruby",
+                        "url": "/questions/1/choices/4",
+                        "votes": 256
+                    }
+                ]
+            }
+```
 
-    ### Unstar a Gist [DELETE]
-    + Response 204
+#### Response without a body
 
-    ### Check if a Gist is Starred [GET]
-    + Response 200
+You can describe a response without a body, for example a 204 response when we delete our question:
 
-        [Star][]
+```
+### Delete [DELETE]
++ Response 204
+```
 
 ## Complete Blueprint
-You can find the complete Gist Fox API blueprint [full listing][] in the [API Blueprint Examples][] repository.
 
-## API Blueprint Tools
-With Gist Fox Blueprint completed it is time to put it at work. For the start you can browse this API [rendered by GitHub][] or enjoy it [rendered by Apiary][].
+You can find the complete Poll API Blueprint [full listing][] in the [API Blueprint Examples][] repository.
 
-Visit the [Tooling Section][] of the [apiblueprint.org][] to find more about tools you can use with your blueprint.
-
----
-
-<a name="indentation"></a>
-### Note on Indentation
-In API Blueprint payload assets are represented by [Markdown pre-formatted code blocks][]. This means **an asset has to be indented by four spaces** relative to its existing indentation level. In the case of Markdown list items, which are also (by definition) indented, it is even one more level further making it a total of **eight spaces** or whopping twelve spaces for list items nested under another list item. 
-
-For example, the asset in `Body` list item which is nested under the `Response 200` item is indented by twelve spaces.
-
-You can save one level of indentation using the [GitHub-flavored Markdown][] syntax for [fenced code blocks][]:
+## API Blueprint Tools
 
-    + Response 200 (application/hal+json)
-        + Headers
+With Poll Blueprint completed it is time to put it at work. For the start you can browse this API [rendered by GitHub][] or enjoy it [rendered by Apiary][].
```

Comment:
I would elaborate here on additional resources. Also the name "API Blueprint Basic Tutorial" implies there is an advanced one...